pysus/online_data/IBGE.py

```diff
@@ -288,3 +296,70 @@ def get_legacy_session():
     session = requests.session()
     session.mount('https://', CustomHttpAdapter(ctx))
     return session
+
+
+def get_population(
+    year: int,
+    source: Literal["POP", "censo", "POPTCU", "projpop"] = "POPTCU",
+    censo_data: Literal["ALF", "ESCA", "ESCB", "IDOSO", "RENDA"] = "ALF"
+) -> pd.DataFrame:
+    """
+    Get population data from IBGE as shared by DATASUS
+    :param year: year of the data
+    :param source: 
```

Comment:
@fccoelho could you please extra check the parameters' `source` and `censo_data` descriptions?

Reply:
They look correct to me.